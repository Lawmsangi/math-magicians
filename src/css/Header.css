.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 5px 20px;
  background-color: #eee;
  height: 10vh;
}

h1 {
  font-size: 40px;
  font-family: 'Ubuntu', sans-serif;
}

nav {
  display: flex;
  gap: 50px;
}

.links {
  text-decoration: none;
  color: #1d1d1d;
  font-size: 25px;
  font-family: 'Ubuntu', sans-serif;
}

.links:hover {
  color: #5e5e5e;
}
