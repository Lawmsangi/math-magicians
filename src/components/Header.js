import React from 'react';
import { Link } from 'react-router-dom';
import '../css/Header.css';

export default function Header() {
  return (
    <div className="header">
      <h1>Math Magicians</h1>
      <nav>
        <Link className="links" to="/">Home</Link>
        <Link className="links" to="/calculator">Calculator</Link>
        <Link className="links" to="/quote">Quote</Link>
      </nav>
    </div>
  );
}
