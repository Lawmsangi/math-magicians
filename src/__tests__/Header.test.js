import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom'; // Import MemoryRouter

import Header from '../components/Header';

test('Header component matches snapshot', () => {
  const { asFragment } = render(
    // Wrap your component with MemoryRouter
    <MemoryRouter>
      <Header />
    </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
