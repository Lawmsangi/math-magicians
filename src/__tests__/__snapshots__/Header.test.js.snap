// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1>
      Math Magicians
    </h1>
    <nav>
      <a
        class="links"
        href="/"
      >
        Home
      </a>
      <a
        class="links"
        href="/calculator"
      >
        Calculator
      </a>
      <a
        class="links"
        href="/quote"
      >
        Quote
      </a>
    </nav>
  </div>
</DocumentFragment>
`;
